components/pixels/lib.rs

```diff
@@ -284,8 +284,7 @@ pub struct RasterImage {
     pub format: PixelFormat,
     pub id: Option<ImageKey>,
     pub cors_status: CorsStatus,
-    #[conditional_malloc_size_of]
-    pub bytes: Arc<IpcSharedMemory>,
+    pub index: ImageByteStorageIndex,
```

Comment:
This data structure implements `Serialize` and `Deserialize`, which will mean that the bytes are serialized and deserialized by serde. This traditionally tanks performance, so I think we cannot do it. It needs to be send via some kind of shared memory or not send across processes.

You might not notice this if the channel isn't an IPC channel, but they always are in multiprocess mode.